stream_alert/rule_processor/rules_engine.py

```diff
@@ -342,25 +342,24 @@ def process(self, input_payload):
         and the rule itself to determine if a match occurs.
 
         Returns:
-            list: alerts
-
-            An alert is represented as a dictionary with the following keys:
-                rule_name: the name of the triggered rule
-                payload: the StreamPayload object
-                outputs: list of outputs to send to
+            (list): First return is a list of alerts.
```

Comment:
For the `Returns` blocks, the parenthesis should not be included around the return type. Please change the back the `list: ...`